generators/generator-bot-core-assistant/README.md

```diff
@@ -1,48 +1,34 @@
 # @microsoft/generator-bot-core-assistant [![NPM version][npm-image]][npm-url]
 
-This template creates an assistant-style conversational bot. Assistant-style bots typically help their users accomplish multiple different tasks, and have support for more varied conversational interactions.
+A bot with Azure Language Understanding (LUIS) and common trigger phrases used to direct the conversation flow and help customers accomplish basic tasks.
```

Comment:
nit: Should we also end with something like "accomplish basic tasks like greetings and QnA."?

Reply:
using language from the design team here.  going with what we have.  we can always refine as needed.